ipinfo/cmd_logout.go

```diff
@@ -35,7 +35,8 @@ func cmdLogout() error {
 	}
 
 	// delete but don't return an error; just log it.
-	if err := deleteToken(); err != nil {
+	gConfig.Token = ""
+	if err := SaveConfig(gConfig); err != nil {
 		fmt.Println("not logged in")
```

Comment:
This change also changes the semantics here; before, it would print "not logged in" if the token was never saved, but now it prints that only if the SaveConfig fails, which it usually will not. We should only print it if `gConfig.Token` was already an empty string before we forcefully made it into an empty string.